default/python/AI/empire/README.md

```diff
@@ -0,0 +1,47 @@
+# Empire info module
+
+This module contains code that responsible for holding useful information about empire objects.
+
+## Implementation details. 
+
+Files in this module wildly use encapsulation principle.
+
+Data container itself is hidden from outer scope and access to it should be provided via functions.
+This approach allows changing internal implementation and add new functionality 
+without changing code which uses this module. 
+Also, we could keуp this module as simple as required for current code.
```

Comment:
`keyp` -> `keep`

Not sure what this sentence means, though.

Reply:
I try to say that we could implement the only API required for current AI code and later add more methods and change the inner structure to support them. 

For example, I have changed `colonies_under_attack` from a list of colonies to bool. It is easy to change it back when we will need it.  And at the current state, this is the simplest possible implementation.